Examples/ReqImport.py

```diff
@@ -26,4 +26,4 @@ def import_req(csvFilePath, specification_ID):
 
 
 
-import_req(csvFilePath, specification_ID)
\ No newline at end of file
+import_req(csvFilePath, specification_ID)SSSS
```

Comment:
Is this a typo?

Reply:
Yes it is a typo, solving it right away.